### PR TITLE
NexCloud changes name to nexclipper-agent

### PR DESCRIPTION
**Pull request by ssipflow**
We have changed its name and convert the full application to saas agent.

### CODE REVIEW

File: repo/packages/N/nexclipper-agent/0/resource.json [hunk 1]
```diff
@@ -1,18 +1,16 @@
 {
   "images": {
```
Comment (joerg84) [inline repo/packages/N/nexclipper-agent/0/resource.json:2]: We shouldn't delete from the universe. Could you leave the old package and just increase the min dcos version to 999 as done here: https://github.com/mesosphere/universe/blob/b5b257e7f92f7e647ae7e90993fdc880f466d7d1/repo/packages/D/dse/0/package.json#L6

**Review by joerg84 — CHANGES_REQUESTED** [review]

File: repo/packages/N/nexclipper-agent/0/resource.json (hunk 1 above)
Comment (joerg84) [reply repo/packages/N/nexclipper-agent/0/resource.json:2]: P.S. I believe that is also causing the one test failure below

Comment (ssipflow) [reply repo/packages/N/nexclipper-agent/0/resource.json:2]: OK I'll leave the old version and convert current version's minDcosReleaseVersion to 999. Thank you

**Comment by ssipflow** [comment]
Hi. I've just leave the old nexcloud package and changed its
minDcosReleaseVersion to 999.
Please checkout the PR #2018.
Thank you

2018년 10월 31일 (수) 오후 2:04, Joerg Schad <notifications@github.com>님이 작성:

**Comment by ssipflow** [comment]
I've just left the old nexcloud package and changed its minDcosReleaseVersion to 999.
Please check out the PR.
Thank you

File: repo/packages/N/nexclipper-agent/0/resource.json (hunk 1 above)
Comment (joerg84) [reply repo/packages/N/nexclipper-agent/0/resource.json]: convert the old  version to 999 ;)

**Review by joerg84 — APPROVED** [review]

**Review by joerg84 — APPROVED** [review]

File: repo/packages/N/nexclipper-agent/0/resource.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "icon-medium": "https://raw.githubusercontent.com/nexclouding/NexCloud/master/imgs/nexclipper-agent-icon-medium.png",
+    "icon-large": "https://raw.githubusercontent.com/nexclouding/NexCloud/master/imgs/nexclipper-agent-icon-large.png",
+    "screenshots": [
+      "https://raw.githubusercontent.com/nexclouding/NexCloud/master/imgs/dashboard.PNG"
+    ]
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "nexclipper-agent": "nexclipper/nexagent:latest"
```
Comment (joerg84) [inline repo/packages/N/nexclipper-agent/0/resource.json]: latest is not the best tag as it is unstable

**Comment by joerg84** [comment]
@ssipflow Sorry I saw one more thing, the docker image tag is latest which is unstable. Could we have a dedicated tag here?

**Comment by ssipflow** [comment]
Sorry, It is a typo. I've removed the tag.

**Comment by joerg84** [comment]
@ssipflow removing is the same as latest, hence will change if you push a new image.
Could you create a new tag (docker tag ...) e.g., with `....:v1.0.0`?

**Comment by ssipflow** [comment]
@joerg84 OK, we have pushed a new image and added new tag ```...:dcos```

**Review by joerg84 — APPROVED** [review]

**Pull request merged**